packages/@webex/plugin-meetings/test/unit/spec/reconnection-manager/index.js

```diff
@@ -10,7 +11,74 @@ const {assert} = chai;
 chai.use(chaiAsPromised);
 sinon.assert.expose(chai.assert, {prefix: ''});
 
+
 describe('plugin-meetings', () => {
+  describe('ReconnectionManager.reconnectMedia', () => {
+    it('uses correct TURN TLS information on reInitiatePeerconnection', async () => {
+      const fakeMediaConnection = {
+        initiateOffer: sinon.stub().resolves({})
```

Comment:
this is not needed. Media connection on master doesn't have initiateOffer() method